excalidraw-app/App.tsx

```diff
@@ -681,19 +684,10 @@ const ExcalidrawWrapper = () => {
   // cases where it still happens, and while we disallow self-embedding
   // by not whitelisting our own origin, this serves as an additional guard
   if (isSelfEmbedding) {
-    return (
-      <div
-        style={{
-          display: "flex",
-          alignItems: "center",
-          justifyContent: "center",
-          textAlign: "center",
-          height: "100%",
-        }}
-      >
-        <h1>I'm not a pretzel!</h1>
-      </div>
-    );
+    // Sparkwise note: Any one-time automatic reload will count as "self-embedding".
+    // In our case, we are not self-embedding, and any reloads that may happen should
+    // not be considered self-embedding and sould not block the user from using the app.
+    posthog.capture("Self-embedding detected");
```

Comment:
Would you want to capture the `currentUrl` value? If so, this could be `posthog.capture("Self-embedding detected", {"origin": new URL(window.location.href).origin});`